<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>805</width>
    <height>479</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="line_command">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>301</width>
      <height>21</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QGroupBox" name="pile">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>70</y>
      <width>121</width>
      <height>191</height>
     </rect>
    </property>
    <property name="title">
     <string>Pile</string>
    </property>
    <widget class="QListWidget" name="listWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>101</width>
       <height>161</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="fonction_pile">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>280</y>
      <width>301</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>Fonction pour Pile</string>
    </property>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>251</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="clear">
        <property name="text">
         <string>clear</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="dup">
        <property name="text">
         <string>dup</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="drop">
        <property name="text">
         <string>DROP</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>80</y>
       <width>239</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="mean">
        <property name="text">
         <string>mean</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="swap">
        <property name="text">
         <string>swap</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="sum">
        <property name="text">
         <string>SUM</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="confirmer">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>10</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Confirmer</string>
     </property>
    </widget>
   </widget>
   <widget class="QLineEdit" name="Result">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>20</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="result">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>20</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Resultï¼š</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="keyboard">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>371</width>
      <height>321</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>323</width>
       <height>257</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="sizeConstraint">
       <enum>QLayout::SetMinAndMaxSize</enum>
      </property>
      <property name="horizontalSpacing">
       <number>7</number>
      </property>
      <property name="margin">
       <number>1</number>
      </property>
      <item row="2" column="0">
       <widget class="QPushButton" name="num4">
        <property name="text">
         <string>4</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="num5">
        <property name="text">
         <string>5</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="num1">
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="num2">
        <property name="text">
         <string>2</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QPushButton" name="num3">
        <property name="text">
         <string>3</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="num6">
        <property name="text">
         <string>6</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="num8">
        <property name="text">
         <string>8</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="num7">
        <property name="text">
         <string>7</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="num9">
        <property name="text">
         <string>9</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QPushButton" name="plus">
        <property name="text">
         <string>+</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QPushButton" name="minus">
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QPushButton" name="multi">
        <property name="text">
         <string>*</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QPushButton" name="POW">
        <property name="text">
         <string>POW</string>
        </property>
       </widget>
      </item>
      <item row="7" column="3">
       <widget class="QPushButton" name="comma">
        <property name="text">
         <string>,</string>
        </property>
       </widget>
      </item>
      <item row="9" column="1">
       <widget class="QPushButton" name="eval">
        <property name="text">
         <string>Eval</string>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QPushButton" name="entrer">
        <property name="text">
         <string>entrer</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QPushButton" name="del">
        <property name="text">
         <string>del</string>
        </property>
       </widget>
      </item>
      <item row="9" column="3">
       <widget class="QPushButton" name="dollar">
        <property name="text">
         <string>$</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QPushButton" name="MOD">
        <property name="text">
         <string>MOD</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QPushButton" name="annuler">
        <property name="text">
         <string>annuler</string>
        </property>
       </widget>
      </item>
      <item row="9" column="2">
       <widget class="QPushButton" name="retablir">
        <property name="text">
         <string>retablir</string>
        </property>
       </widget>
      </item>
      <item row="8" column="2">
       <widget class="QPushButton" name="SIGN">
        <property name="text">
         <string>SIGN</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QPushButton" name="num0">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1" colspan="2">
       <widget class="QPushButton" name="espace">
        <property name="text">
         <string>Space</string>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="QPushButton" name="slash">
        <property name="text">
         <string>/</string>
        </property>
       </widget>
      </item>
      <item row="7" column="2">
       <widget class="QPushButton" name="guillemet">
        <property name="text">
         <string>'</string>
        </property>
       </widget>
      </item>
      <item row="8" column="3">
       <widget class="QPushButton" name="point">
        <property name="text">
         <string>.</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="Fonction_unaire">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>60</y>
      <width>181</width>
      <height>201</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QWidget" name="gridLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>158</width>
       <height>191</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QPushButton" name="SIN">
        <property name="text">
         <string>SIN</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="COS">
        <property name="text">
         <string>COS</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="TAN">
        <property name="text">
         <string>TAN</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="SIHN">
        <property name="text">
         <string>SINH</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="LN">
        <property name="text">
         <string>LN</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QPushButton" name="INV">
        <property name="text">
         <string>INV</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QPushButton" name="SQRT">
        <property name="text">
         <string>SQRT</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QPushButton" name="SQR">
        <property name="text">
         <string>SQR</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="TANH">
        <property name="text">
         <string>TANH</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="COSH">
        <property name="text">
         <string>COSH</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="LOG">
        <property name="text">
         <string>LOG</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QPushButton" name="CUBE">
        <property name="text">
         <string>CUBE</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>805</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOuvrir_la_source"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="action"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="action">
   <property name="text">
    <string>?</string>
   </property>
  </action>
  <action name="actionOuvrir_la_source">
   <property name="text">
    <string>Ouvrir la source</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>404</x>
     <y>216</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>del</sender>
   <signal>clicked()</signal>
   <receiver>line_command</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>231</x>
     <y>278</y>
    </hint>
    <hint type="destinationlabel">
     <x>200</x>
     <y>65</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
